libraries/bxdf/standard_surface.mtlx

```diff
@@ -129,11 +129,12 @@
     <normalize name="tangent_rotate_normalize" type="vector3">
       <input name="in" type="vector3" nodename="tangent_rotate" />
     </normalize>
-    <compare name="main_tangent" type="vector3">
-      <input name="intest" type="float" interfacename="specular_anisotropy" />
-      <input name="in1" type="vector3" interfacename="tangent" />
-      <input name="in2" type="vector3" nodename="tangent_rotate_normalize" />
-    </compare>
+    <ifgreater name="main_tangent" type="vector3">
```

Comment:
@niklasharrysson , Can you double check I have the new logic correct. I think it's good now.